english/I'm-batman/batman.md

```diff
@@ -5,4 +5,6 @@ Be careful of the Joker!
 [Realize that you are Batman and have the Utility Belt on you. Yeah!!!](https://youtu.be/YpIQQeL2ZYk?t=22s)
 
 Alfred Pennyworth: Why do we fall sir? So we might learn to pick ourselves up.
-Bruce Wayne: You still haven't given up on [me](https://www.youtube.com/watch?v=u843KNE-exo)? 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.